tests/test_spec2vec_client.py

```diff
@@ -6,6 +6,12 @@
 from omigami.spec2vec import InvalidCredentials
 
 
+@pytest.fixture()
+def positive_ion_endpoint():
+    _client = Spec2Vec("")
+    return _client._PREDICT_ENDPOINT_BASE.format(ion_mode="positive")
+
+
```

Comment:
Why can't you define this as a function so we can get the **positive** and the **negative** endpoint?